Woof/WoofTests/NetworkLayer/Endpoint/WoofAppEndpointTests.swift

```diff
@@ -0,0 +1,43 @@
+import NetworkService
+import XCTest
+
+final class WoofAppEndpointTests: XCTestCase {
+    func testAllWoofAppEndpointPropertiesAreAvailable() {
+        let _ = WoofAppEndpoint.networkEnvironment
+    }
+
+    func testAllCasesForWoofAppEndpointExists() {
+        let _ = WoofAppEndpoint.addNewSitter(Sitter())
+    }
+
+    func testAddNewSitterEndpointIsConfiguredCorrectly() {
+        // Given
+        let baseProdURL = URL(string: "https://woof-app.hasura.app/api/rest/") ?? Bundle.main.bundleURL
+        let authHeader = "x-hasura-admin-secret"
+
+        // When
+        let sitter = Sitter()
+
+        let endpoint = WoofAppEndpoint.addNewSitter(sitter)
+
+        let method = endpoint.method
+        let task = endpoint.task
+        let headers = endpoint.headers
+
+        // Then
+        XCTAssertEqual(endpoint.baseURL, baseProdURL)
+        XCTAssertEqual(endpoint.path, EndpointPath.addNewSitter)
+        XCTAssertTrue(headers.keys.contains(authHeader))
+
+        switch method {
+        case .post: break
+        default: XCTFail("Wrong method: expected `POST` method, but \(method) was settled")
+        }
+
+        switch task {
+        // swiftlint:disable:next empty_enum_arguments
+        case .requestWithBodyParameters(_): break
+        default: XCTFail("Wrong task type: expected `requestWithBodyParameters` task, but \(task) was settled")
+        }
```

Comment:
```suggestion
        let endpoint = WoofAppEndpoint.addNewSitter(sitter)

        // Then
        XCTAssertEqual(endpoint.baseURL, baseProdURL)
        XCTAssertEqual(endpoint.path, EndpointPath.addNewSitter)
        XCTAssertTrue(endpoint.headers.keys.contains(authHeader))
        
        switch endpoint.method {
        case .post: break
        default: XCTFail("Unexpected scenario.")
        }

        switch endpoint.task {
        case .requestWithBodyParameters: break
        default: XCTFail("Unexpected scenario.")
        }
```
I propose to simplify. 

- don't create separate constants because we can use the dot syntax here and that is not tangled.
- we don't really need to bind values which triggered an error, because we can access them without additional efforts in tests.

Reply:
Agree, your version is more readable. 👍

Reply:
Change to your version, I thought that it could be useful if we run tests in CI the info displayed in the log file will be more detailed and useful.